AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLModelBasedTests.swift

```diff
@@ -294,6 +295,54 @@ class GraphQLModelBasedTests: XCTestCase {
         wait(for: [completeInvoked], timeout: TestCommonConstants.networkTimeout)
     }
 
+    func testUpdatePostWithInvalidCondition() {
```

Comment:
`testUpdatePostWithInvalidConditionShouldReturnConditionalCheckFailed()`

Reply:
Well, this is a long name😂